### DISCUSSION TITLE
Entire client is suddenly transparent

### DISCUSSION THREAD

**Discussion by Mysticthundr** [runelite/runelite#17455] [Q&A] [answered]
Hey all, looking for some help here. I tried to log in today for the first time in about 3 or 4 days, and my entire client is suddenly transparent. Looking for some help in what might have changed. I'm on a Macbook Pro running Sonoma 14.2.1 (23C71). Restarted the laptop, deleted and reinstalled the client, can't find any settings anywhere that might have changed.

<img width="1512" alt="Screenshot 2024-02-05 at 10 11 54 PM" src="https://github.com/runelite/runelite/assets/159098111/d3a8505a-eb5c-4fd9-a536-7f39bcf3c8f9">

**Comment by Felanbird** [comment, marked as answer]
Click the 🔧 near the top right, search for the `RuneLite` plugin and inside that you should see an opacity setting that's probably set to `10`, set it to `100`

This feature previously did not work on macOS/Linux, now it does.

**Reply by Mysticthundr** [reply]
That was an easy fix. Thanks so much @Felanbird!
